examples/exec/Exec.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Threading.Tasks;
+using k8s;
+using k8s.Models;
+
+namespace exec
+{
+    internal class Exec
+    {
+        private static void Main(string[] args)

```

Comment:
Note sure what version of C# this is built with but latest supports async main if that makes the code cleaner?

Reply:
Done (had to move to c# 7.1)